PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/config/FunctionAppConfiguration.kt

```diff
@@ -38,6 +38,10 @@ import org.jdom.Element
 class FunctionAppConfiguration(project: Project, factory: ConfigurationFactory, name: String?) :
         AzureRunConfigurationBase<FunctionAppSettingModel>(project, factory, name) {
 
+    companion object {
+        private const val SUGGESTED_NAME_PREFIX = "Publish Azure Function App"
```

Comment:
Do we want to have this long generated names for each run configuration? Up to your judgment :) : I would suggest to truncate it to "Publish - <project_name>". All run configurations are collected under "Azure - Publish Web App" group. So, it already makes sense that we will publish to web apps.

Reply:
Made them shorter now :-)